javascript/javascript1/week1/ezNamey.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hack Your Future, Javascript, homework Week1
+ * this file includes:
+  4. Ez Namey (Startup name generator) 
+*/
+
+
+// 4. Ez Namey (Startup name generator)
+
+const firstWords = ["Easy", "Awesome", "Corporate", "Amazing", "Helpful", "Perfect", "Attractive", "Excellent", "onderful", "Incredible"]
+const secondWords = ["Company", "Firm", "Agency", "Group", "Business", "Team", "Corporation", "Partnership", "Institution", "Association"]
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
+
+let startupName = firstWords[randomNumber]+ " " + secondWords[randomNumber];
+
+console.log("The startup: " + `"` + startupName +`"` + " contains " + startupName.length + " characters" )
```

Comment:
Good to see that you use ``(ticks). 
Try to wrap whole console.log("The startup: " + `"` + startupName +`"` + " contains " + startupName.length + " characters" ) statementusing ticks.